packages/caliper-tests-integration/fabric_tests/run.sh

```diff
@@ -29,10 +29,13 @@ cd ${DIR}
 
 # bind during CI tests, using the package dir as CWD
 # Note: do not use env variables for binding settings, as subsequent launch calls will pick them up and bind again
+FABRIC_VERSION=1.4.20
```

Comment:
can we leave this a 1.4 ? then it will always test the latest 1.4 version without any changes to the integration tests

Reply:
The problem with leaving it at latest is how do we invalidate the cache. Even when a new version is released (like 1.4.21), our CI pipeline will still be using the older cached version, giving a false impression that the tests are passing against the latest Fabric 1.4. If we were explicit about the versions used, I think it would be easy to spot if later versions cause issues with the codebase.

Reply:
So the cache is persistent across actions then ? if so then I see what you mean. However we then have a problem with 2.4 as we don't have a version number for it like 1.4.20, 2.2.14. 
It would be a bit of a pain to update the version number in caliper and remember to also upgrade the integrations tests and that was what we wanted to avoid.
1.4.20 and 2.2.14 aren't likely to be increased now, so maybe could we not cache the 2.4 one. the bind of 2.4 would be quite quick compared to 1.4 and 2.2 as 2.4 is very light weight

Reply:
Yes. I have taken the versions from https://github.com/hyperledger/caliper/blob/main/packages/caliper-cli/lib/lib/config.yaml, however there isn't a minor version for 2.4 as of now, so I had to leave it as 2.4. Should we add a minor version in the config file or could I just remove the caching for 2.4 (I think removing caching for just 2.4 is going to be hacky)?

Reply:
I was thinking of extracting the version numbers out of the script and passing it through the workflow, however I was thinking about two issues that it might raise:
1. The Fabric test script itself uses 3 versions of Fabric, so it would be a challenge to pass that to the script.
2. Even if we extract the version numbers to environment variables in the workflow, we still need to hard-code the version numbers in the script that developers use to run tests. So, that would only increase the number of places we need to change to update the version numbers.

Reply:
Also lets add some comments in explaining this ie what's cached and not cached and the use of the detailed version numbers in this file plus we should add comments to `packages\caliper-cli\lib\lib\config.yaml` as well otherwise I can see that this could get missed if the config.yaml file was updated but entries in the integration tests weren't